### ISSUE TITLE
`glide tree` command doesn't exclude core libraries

### ISSUE THREAD

**Issue by janeczku** [Masterminds/glide#76]
### Steps to reproduce

1. Build current HEAD of glide
2. Unset $GOROOT environment variable
3. Run  `glide tree`

### Expected result

 `glide tree` prints a tree of imports **excluding** core libraries

### Actual result
`glide tree` prints out dozens of lines with core libraries

```sh
~/go/src/github.com/janeczku/go-dnsmasq$ glide tree
github.com/janeczku/go-dnsmasq
	fmt	(glide get fmt)
	github.com/codegangsta/cli   (/Users/Janek/go/src/github.com/codegangsta/cli)
	errors	(glide get errors)
	flag	(glide get flag)
	fmt	(glide get fmt)
	io	(glide get io)
	io/ioutil	(glide get io/ioutil)
	os	(glide get os)
(...) 400 more lines like that
```

### Additional information

This seems related to #74.    
The difference is that this affects all systems that don't have $GOROOT set (not just those that run a glide binary that was compiled on a different flavor of Go installation)

**Comment by technosophos** [comment]
Yeah, you're right. Definitely related to #74 and #75. The `Goroot` flag on the imports of core packages should have been set to `true`, but with no `$GOROOT`, it doesn't get set correctly.

**Comment by mattfarina** [comment]
@janeczku can you try the latest (0.6.0) to see if this is fixed.

**Comment by janeczku** [comment]
@mattfarina this is fixed (checked with 0.6.0)

**Comment by mattfarina** [comment]
Thanks
